provers/sp1/guest/src/zk_op.rs

```diff
@@ -27,7 +27,7 @@ impl ZkvmOperator for Sp1Operator {
             .map(|chunk| u32::from_le_bytes(chunk.try_into().unwrap()))
             .collect::<Vec<u32>>()
             .try_into()
-            .map_err(|e| Error::ZkvmOperation("Input point processing failed".to_string()))?;
+            .map_err(|_| Error::ZkvmOperation("Input point processing failed: {}".to_string()))?;
```

Comment:
Shouldn't this take the `e` argument and use it within the `format!` macro like:
```rust
.map_err(|e| Error::ZkvmOperation(format!("Input point processing failed: {e}")))?;
```

Reply:
tried that, but, apparently, for some targets `e` is `Vec<u8>`, which might be just underlying string bytes or it might have some sensitive info leaked. Thus stayed with current approach of ignoring it but suppressed warning.

Reply:
Than we can just ignore `e` but also remove the `: {}` that's added with the change

Reply:
sure. missed that one.